package.json

```diff
@@ -45,6 +45,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "4.5.0",
     "ember-source": "~3.0.0",
+    "ember-truth-helpers": "2.0.0",
```

Comment:
Don't add this dependency, just to use the `(and... )` in your hbs. You can make a computed property